vm/runtime.ts

```diff
@@ -1467,12 +1482,8 @@ module J2ME {
       }
     }
 
-    if (false && methodTimeline) {
-      fn = profilingWrapper(fn, methodInfo, methodType);
-    }
-
-    if (traceWriter) {
-      fn = tracingWrapper(fn, methodInfo, methodType);
+    if (profile) {
+      fn = wrapMethod(fn, methodInfo, methodType);
```

Comment:
It looks like this disables the trace writer unless profiling is enabled, but I find the trace writer to be handy even without the profiler, so I'd rather remain able to enable it discretely.